meta/aspell.en.pws

```diff
@@ -38,6 +38,7 @@ ffff
 FPGAs
 fprintf
 functionalities
+FX
```

Comment:
could be in acronym as full duplex ?

Reply:
Looks like you don't need this change as you approved it.